dist/SectionsContainer.js

```diff
@@ -98,6 +98,17 @@ var SectionsContainer = _react2['default'].createClass({
     this._addMouseWheelEventHandlers();
   },
 
+  _loadLazySrc: function _loadLazySrc() {
+    var imgs = document.querySelectorAll(".container > div")[this.state.activeSection].querySelectorAll('img');
```

Comment:
document.querySelectorAll is not a react-way.
Dont do that.
Why not just use [react-lazy-load](https://www.npmjs.com/package/react-lazy-load) component?